src/consumption.cpp

```diff
@@ -1093,9 +1093,19 @@ static bool eat( item &food, Character &you, bool force )
 void Character::modify_health( const islot_comestible &comest )
 {
     const int effective_health = comest.healthy;
-    // Effectively no cap on health modifiers from food and meds
-    const int health_cap = 200;
-    mod_daily_health( effective_health, effective_health >= 0 ? health_cap : -health_cap );
+    int daily_food_health += effective_health;
```

Comment:
Seems like you're trying to... increment a declaration? I don't think this is what you meant to do. I think you mean for this to be stored as a variable in the character, which is saved and loaded with the game.

Reply:
Most likely. I don't recall quite what I was doing but I can see that this doesn't do what it's supposed to.